openassessment/xblock/static/js/src/studio/oa_edit_settings.js

```diff
@@ -82,6 +95,16 @@ export class EditSettingsView {
       { min: 0, max: 100 },
     );
 
+    this.resubmissionsGracePeriodHoursIntField = new IntField(
+      $('#openassessment_resubmissions_grace_period_hours', this.element),
+      { min: 0, max: 59 },
```

Comment:
Question: why is the max for hours 59? shouldn't these limits be set to the minute's fields?

Reply:
You are right, this limit is for minutes, not hours, I will correct it.